notebooks/Binary Classification with the UCI Credit-card Default Dataset.ipynb

```diff
@@ -29,31 +30,40 @@
     "## What is Covered\n",
     "\n",
     "* **Domain:**\n",
-    "  * Finance (loan decisions). The data is semisynthetic to create a simple example of disparity in accuracy.\n",
+    "  * Finance (loan decisions). The data is semisynthetic to create a simple example of disparities in FPR and FNR.\n",
     "\n",
     "* **ML task:**\n",
     "  * Binary classification.\n",
     "\n",
     "* **Fairness tasks:**\n",
-    "  * Assessment of unfairness using Fairlearn metrics and Fairlearn dashboard.\n",
+    "  * Assessment of unfairness using Fairlearn metrics.\n",
     "  * Mitigation of unfairness using Fairlearn mitigation algorithms.\n",
     "\n",
     "* **Performance metrics:**\n",
     "  * Area under ROC curve.\n",
     "  * Balanced accuracy.\n",
     "\n",
     "* **Fairness metrics:**\n",
+    "  * False-positive rate difference.\n",
+    "  * False-negative rate difference.\n",
     "  * Equalized-odds difference.\n",
     "\n",
     "* **Mitigation algorithms:**\n",
     "  * `fairlearn.reductions.GridSearch`\n",
-    "  * `fairlearn.postprocessing.ThresholdOptimizer`\n",
-    "\n",
+    "  * `fairlearn.postprocessing.ThresholdOptimizer`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "## Introduction\n",
     "\n",
-    "In this example, we emulate the problem of accuracy disparities arising in loan decisions. Specifically, we consider scenarios where algorithmic tools are trained on historic data and their predictions about loan applicants are used for making decisions about applicants. See [here](https://www.nytimes.com/2019/11/10/business/Apple-credit-card-investigation.html) for an example involving sex-based discrimination for credit limit decisions.  \n",
+    "In this notebook, we consider a scenario where algorithmic tools are deployed to predict the likelihood that an applicant will default on a credit-card loan. The notebook emulates the problem presented in this [white paper](https://www.microsoft.com/en-us/research/uploads/prod/2020/09/Fairlearn-EY_WhitePaper-2020-09-22.pdf) in collaboration with EY.\n",
+    "\n",
+    "Due to data privacy, we do not use the data from the white paper. Instead, we use the [UCI Credit-card default dataset](https://archive.ics.uci.edu/ml/datasets/default+of+credit+card+clients), a toy dataset reflecting credit-card defaults in Taiwan, as a substitute dataset to replicate the desired workflow. To make this dataset applicable to our problem, we introduce a synthethic feature that is highly predictive for applicants defined as \"female\" in terms of the \"sex\" feature, but is uninformative for applicants defined as \"male\".\n",
```

Comment:
Is it really a "toy" dataset? It seems to be real one. It may also be worth mentioning that one shouldn't assume that a dataset from a one context will be suitable for another.